semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/LiteralMethods.scala

```diff
@@ -1,20 +1,60 @@
 package io.shiftleft.semanticcpg.language.nodemethods
 
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.Languages
 import io.shiftleft.codepropertygraph.generated.nodes.{Literal, NewLocation}
 import io.shiftleft.semanticcpg.NodeExtension
 import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.semanticcpg.language.nodemethods.LiteralMethods.delimiters
+import org.apache.commons.lang3.StringUtils
+
+import scala.collection.immutable.HashMap
 
 class LiteralMethods(val literal: Literal) extends AnyVal with NodeExtension with HasLocation {
-  def value(typeFullName: String): String = {
-    if (literal.typeFullName == typeFullName) {
-      return "asd"
+  def innerText: Option[String] = {
+    val stringDelimiter = Cpg(literal.graph).metaData.language.headOption match {
+      case Some(language) => delimiters.getOrElse(language, List("\""))
+      case _              => "\"" :: Nil
     }
 
-    return ""
+    stringDelimiter
+      .filter(literal.code.startsWith(_))
+      .map(delimiter =>
+        val start =
+          if (delimiter == "\"\"\"" || delimiter == "'''") then literal.code.indexOf(delimiter) + 3
+          else literal.code.indexOf(delimiter) + 1
+        val end = literal.code.lastIndexOf(delimiter)
+
+        /* Windows uses `\r\n` for new line characters, but this can cause issues on different operating systems. So
+        we replace all `\r\n` to have consistency across all OS
+         */
+        literal.code.substring(start, end).replaceAll("\r\n", "\n")
```

Comment:
If this causes issues why do we address this in such a special purpose function as this one?